Automattic/wp-calypso
# Untangling: Clean up layout/{dotcom-nav-redesign-v2,wpcom-admin-interface} flags

Fixes https://github.com/Automattic/dotcom-forge/issues/7920

## Proposed Changes

This PR cleans up the following flags:

- `layout/dotcom-nav-redesign-v2`
- `layout/wpcom-admin-interface`

## Why are these changes being made?

Untangling & Nav Redesign has been launched.

## Testing Instructions

1. Open /sites
2. Open a site
3. Smoke test and verify that everything still works.

## Pre-merge Checklist
